Extension/src/LanguageServer/settings.ts

```diff
@@ -564,6 +564,10 @@ export class CppSettings extends Settings {
             && vscode.workspace.getConfiguration("workbench").get<string>("colorTheme") !== "Default High Contrast";
     }
 
+    public get sshTargetsView(): string {
+        return super.Section.get<string>("sshTargetsView") || 'auto';
```

Comment:
I think the `||` should be `??` if the intention is for `auto` to be used when the get returns null or undefined. I think practically it's just a style change though.

Reply:
Sure